internal/pkg/routes/mfx/mfx.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mfx
+
+import (
+	"fmt"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/gogo/protobuf/proto"
+	"github.com/mainflux/export/internal/pkg/routes"
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/senml"
+	"github.com/nats-io/nats.go"
+)
+
+const (
+	// JSON represents SenML in JSON format content type.
+	JSON = "application/senml+json"
```

Comment:
I would use `ContentTypeJSON`.

Reply:
https://github.com/mainflux/export/pull/6#discussion_r365568733

Reply:
https://github.com/mainflux/export/pull/6/files/b503b8c7521d031657d169f0aa17b2d1a47fbb28#discussion_r365568733

Reply:
to call it you will use `mfx.JSON` which is less self explanatory than `mfx.ContentTypeJSON`